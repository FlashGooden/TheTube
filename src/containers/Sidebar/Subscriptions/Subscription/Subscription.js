import React from 'react'
import {Icon,Image,Menu} from 'semantic-ui-react'
import './Subscription.scss'

export function Subscription(props) {
  let rightElement = null
  const {broadcasting, amountNewVideos} = props
  return (
    <div>
      
    </div>
  )
}
