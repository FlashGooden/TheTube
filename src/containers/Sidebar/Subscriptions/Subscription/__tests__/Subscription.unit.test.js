import React from 'react'
import {shallow} from 'enzyme'
import Subscription from '../Subscription'
import {Menu} from 'semantic-ui-react'

test('should render Subscription', ()=> {
  const wrapper = shallow(
    <Menu>
      <Subscription />
    </Menu>
    
  )
  expect(wrapper).toMatchSnapshot()
})