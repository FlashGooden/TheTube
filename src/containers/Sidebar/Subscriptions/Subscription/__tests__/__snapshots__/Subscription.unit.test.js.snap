// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscription Renders Subscription with Broadcasting property 1`] = `
<MenuItem>
  <div
    className="subscription"
  >
    <div>
      <Image
        as="img"
        avatar={true}
        src="http://via.placeholder.com/28x28"
        ui={true}
      />
      <span />
    </div>
    <Icon
      as="i"
      name="signal"
    />
  </div>
</MenuItem>
`;

exports[`Subscription Renders Subscription with video count for label 1`] = `
<MenuItem>
  <div
    className="subscription"
  >
    <div>
      <Image
        as="img"
        avatar={true}
        src="http://via.placeholder.com/28x28"
        ui={true}
      />
      <span>
        Courseara
      </span>
    </div>
    <span
      className="new-videos-count"
    >
      10
    </span>
  </div>
</MenuItem>
`;

exports[`Subscription should render Subscription 1`] = `
<MenuItem>
  <div
    className="subscription"
  >
    <div>
      <Image
        as="img"
        avatar={true}
        src="http://via.placeholder.com/28x28"
        ui={true}
      />
      <span />
    </div>
  </div>
</MenuItem>
`;
